CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068)
 
+### Fixed
+
+- Enable `otelhttp.Handler` to propagate multiple WriteHeader calls while persisting initial statusCode in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#3580)
```

Comment:
```suggestion
- Enable `otelhttp.Handler` to propagate multiple `WriteHeader` calls while persisting the initial HTTP status code in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#3580)
```

Reply:
Thank you so much. I struggled so much finding the words to describe this tiny change.